engine/src/main/java/org/terasology/recording/CharacterStateEventPositionMap.java

```diff
@@ -21,14 +21,26 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * Saves the "position" and "velocity" variable for every CharacterStateEvent so that the player location can be
+ * reproduced correctly during a replay.
+ */
 public class CharacterStateEventPositionMap {
 
+    /** map in which the key is the "sequenceNumber" of the CharacterStateEvent and the value is an array with the
```

Comment:
Trivial: IMHO this could fit in a `/** Some text */` javadoc one liner. Otherwise I'd suggest the usual format:

```
/**
 * Line 1
 * Line 2
 */
```

For consistency more than anything. I like the one liners for fields, you can usually describe them briefly enough to not need more lines - if not it may be too complex anyway :-)